bin/benchpark

```diff
@@ -21,16 +21,24 @@ lndir -silent ${TEST_SOURCE_DIR}/configs/${TEST_CONFIG_NAME} ${TEST_WORKING_DIR}
 git clone --depth=1 -c feature.manyFiles=true https://github.com/spack/spack.git ${TEST_WORKING_DIR}/spack
 git clone --depth=1 -c feature.manyFiles=true https://github.com/GoogleCloudPlatform/ramble.git ${TEST_WORKING_DIR}/ramble
 
+echo CLONED
+
 export SPACK_DISABLE_LOCAL_CONFIG=1
 rm -rf ~/.ramble/repos.yaml
 
+echo REMOVED
```

Comment:
@becker33 are these debug statements intentional or remnants from earlier?

Reply:
Remnants, I thought I'd pulled them out of the final version I pushed